torchbenchmark/e2e_models/hf_bert/__init__.py

```diff
@@ -65,7 +81,11 @@ def __init__(self, test, batch_size=None, extra_args=[]):
     
     def prep(self, hf_args):
         # Initialize the accelerator. We will let the accelerator handle device placement for us in this example.
-        accelerator = Accelerator(fp16=(self.tb_args.fp16 == "amp"))
+        if hasattr(hf_args, "deepspeed_plugin"):
```

Comment:
Similarly, this should be part of `apply_deepspeed()`

Reply:
see my comment above.  If you're ok with rewriting how 'prep' and __init__ works for this model, then i think i can make that work.  But the way it is now, unless I just replace the Accelerator after calling prep (which seems a little bad, esp.  since the accelerators seem to rendezvous in '.wait_for_everyone()'.